src/uct/api/uct.h

```diff
@@ -3044,6 +3032,9 @@ ucs_status_t uct_cm_query(uct_cm_h cm, uct_cm_attr_t *cm_attr);
  * @ingroup UCT_CLIENT_SERVER
  * @brief Create a new transport listener object.
  *
+ * This routine creates a new transport listener on the given CM which
+ * willl start listening on a given sockaddr.
```

Comment:
`will`

Reply:
`This routine creates a new transport listener` ->`This routine creates a new  listener`